stockalert/app.py

```diff
@@ -15,7 +15,11 @@
 
 def read_local_skus() -> List[Sku]:
     with open("skus.csv", encoding='utf-8') as csvf:
-        return [Sku(x) for x in csv.DictReader(csvf)]
+        skus = [Sku(x) for x in csv.DictReader(csvf)]
+    for sku in skus:
+        if sku.price_threshold is not None:
+            sku.price_threshold = float(sku.price_threshold)
```

Comment:
seems like this could have lived in the constructor

Reply:
yea i thought about that but the conversion is a side effect of using the CSV format and i thought it would be nice to put all CSV-related logic in one function